modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/mdf-claim.json

```diff
@@ -7,6 +7,28 @@
 		"en_US": "MDF Claim"
 	},
 	"name": "MDFClaim",
+	"objectActions": [
+		{
+			"active": true,
+			"errorMessage": {
+			},
+			"externalReferenceCode": "AUTOMATION-TO-COMPLETE-REQUEST-ACTION",
+			"label": {
+				"en_US": "Automation To Complete Request"
```

Comment:
```suggestion
				"en_US": "MDF Claim Status Management"
```

Reply:
we want to leave its name generic, so we can use the CE for other Status changes

Reply:
Resolved